src/transformations/trim-lines.js

```diff
@@ -0,0 +1,16 @@
+const consola = require('consola');
+const _ = require('lodash');
+
+/**
+ * This is a very simple transformation that removes leading and trailing spaces/tabs.
+ *
+ * @param {Array<string>} lines - lines/rules to transform
+ * @returns {Array<string>} filtered lines/rules
+ */
+function trimLines(lines) {
+    const transformed = lines.map((line) => _.trim(line, [' ', '\t']));
```

Comment:
@scripthunter7 lodash `_.trim` function for the second argument takes string, not array
with the array, it implicitly trims commas in the lines too, though there are no such cases in the tests